lib/pages/movie_info_screen/movie_details.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import '/constants.dart';
+
+class MovieDetails extends StatelessWidget {
+  String movieTitle;
+  String movieReleaseDate;
+  num movieVoteAverage;
+  static const String releaseDateText = 'Release date: ';
+  static const double movieTitleSize = 36;
+  static const double movieReleaseDateSize = 18;
+  static const double movieVoteAverageSize = 25;
+  static const double starIconSize = 45;
+
+  MovieDetails(this.movieTitle, this.movieReleaseDate, this.movieVoteAverage,
+      {super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          movieTitle,
+          style: const TextStyle(
+            color: Color(
+              AppConstants.appFontColor,
+            ),
+            fontSize: movieTitleSize,
+          ),
+        ),
+        Row(
+          children: [
+            const Icon(
+              Icons.star_half,
+              color: Color(
+                AppConstants.starColor,
+              ),
+              size: starIconSize,
+            ),
+            Text(
+              '$movieVoteAverage',
+              style: const TextStyle(
+                  color: Color(AppConstants.appFontColor),
+                  fontSize: movieVoteAverageSize),
+            ),
+          ],
+        ),
+        Text(
+          '$releaseDateText $movieReleaseDate',
+          style: const TextStyle(
+            color: Color(
+              AppConstants.appFontColor,
+            ),
+            fontSize: movieReleaseDateSize,
+          ),
```

Comment:
I'm seeing this text with the same color a lot, let's make a custom widget with that default color and all the other variables you need customizable, you are going to save a lot of code